code/android-core-library/src/phone/java/com/adobe/marketing/mobile/services/ServiceProvider.java

```diff
@@ -61,6 +64,7 @@ private ServiceProvider() {
         defaultUIService = new AndroidUIService();
         messageDelegate = null;
         defaultLoggingService = new AndroidLoggingService();
+        defaultCacheService = new FileCacheService(defaultDeviceInfoService);
```

Comment:
Can we avoid passing `defaultDeviceInfoService` to `FileCacheService`? It can instead call `ServiceProvider.getDeviceInfoService().getCacheDir()` when needed as our APIs allow deviceInfoService to be overwritten.